serverlessworkflow/modules/ROOT/pages/cloud/quarkus/build-workflow-images-with-tekton.adoc

```diff
@@ -1 +1,167 @@
+You have a git repository containing your workflow and the Containerfile to build the workflow as a container image
+and a namespace where you can install Tekton i.e. `custom-namespace`.
+
+.OptionalPrerequisites
+Tekton CLI:
+
+- On Openshift in the Operator pipelines is available a link to download the correct cli from the operaotr
+- On Minikube/kind/Kubernetes use this link https://tekton.dev/docs/cli/
+
+
+== Install Tekton pipeline
+
+If you are using Openshift, you can install the Red Hat Pipeline Operator.
+If you are using Minikube, you can follow this link:https://github.com/tektoncd/pipeline/blob/main/docs/developers/local-setup.md#using-minikube[tekton guide for Minikube].
+If you are using Kind you can follow this link:https://github.com/tektoncd/plumbing/tree/main/hack[Tekton guide for Kind].
+If you want to use it on a plain kubernetes cluster, you can follow this link:https://tekton.dev/docs/pipelines/install/[Tekton guide for kubernetes].
+
+== Create the pipeline
+
+In the `custom-namespace` namespace, in the pipelines menu on Openshift UI, open the pipeline item, create it, and copy the following `custom-workflow-image-pipeline.yaml` file.
+It contains a pipeline to clone your git repo, build the image and push it to the local image registry.
+
+----
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: custom-workflow-image-pipeline
+  namespace: custom-namespace
+spec:
+  params:
+    - description: name of the deployment to be patched
+      name: deployment-name
+      type: string
+    - description: url of the git repo for the code of deployment
+      name: git-url
+      type: string
+    - default: pipelines-1.9
+      description: revision to be used from repo of the code for deployment
+      name: git-revision
+      type: string
+    - description: image to be built from the code
+      name: IMAGE
+      type: string
+  tasks:
+    - name: fetch-repository
+      params:
+        - name: url
+          value: $(params.git-url)
+        - name: subdirectory
+          value: ''
+        - name: deleteExisting
+          value: 'true'
+        - name: revision
+          value: $(params.git-revision)
+      taskRef:
+        kind: ClusterTask
+        name: git-clone
+      workspaces:
+        - name: output
+          workspace: shared-workspace
+    - name: build-image
+      params:
+        - name: IMAGE
+          value: $(params.IMAGE)
+        - name: TLSVERIFY
+          value: 'false'
+      runAfter:
+        - fetch-repository
+      taskRef:
+        kind: ClusterTask
+        name: buildah
+      workspaces:
+        - name: source
+          workspace: shared-workspace
+  workspaces:
+    - name: shared-workspace
+----
+
+If you use the Tekton Dashboard on Minikube, Kind or on a Kubernetes cluster, copy the content of the yaml in the pipeline section.
+If you don't want to use the UI, replace the placeholder in the `custom-workflow-image-pipeline.yaml` (the previous yaml) and execute:
```

Comment:
instead using "replace placeholder", imho, it is better to use something like, "fill the required fields in the example file above and..."